versioned_docs/version-6.x.x/other-topics/upgrade.md

```diff
@@ -24,7 +24,7 @@ Sequelize.useCLS(namespace);
 
 ### Database Engine Support
 
-We have updated our minimum supported database engine versions. Using older database engine will show `SEQUELIZE0006` deprecation warning. Please check [ENGINE.md](https://github.com/sequelize/sequelize/blob/main/ENGINE.md) for version table.
+We have updated our minimum supported database engine versions. Using older database engine will show `SEQUELIZE0006` deprecation warning. Please check [ENGINE.md](https://github.com/sequelize/sequelize/blob/v6.20.1/ENGINE.md) for version table.
```

Comment:
Can we make this the v6 branch?

Reply:
Let's point it to https://sequelize.org/releases/ instead

Reply:
@ephys done, let me know if the wording is OK

Reply:
Looks good to me! :)